docs/reference/security/sso.md

```diff
@@ -131,7 +129,11 @@ Steps:
 4. Under Configure platforms, select the web option.
 5. Select Configure to complete the platform configuration.
 
-Once you finish registering lakeFS Cloud with Azure AD, save the **Application (Client) ID**, **Application Secret Value** and send this to Treeverse's team to finish the integration.
+Once you finish registering lakeFS Cloud with Azure AD send the following items to the Treeverse's team:
+- **Client ID**
+- **Client Secret**
+- **Azure AD Domain**
+
```

Comment:
We also need the Identity API version they are using.
This can be:

- v1 for Azure AD
- v2 for Microsoft Identity Platform/Entra